src/components/Comments.jsx

```diff
@@ -24,12 +25,14 @@ function Comments() {
   if (error) return <p>No comments found</p>;
```

Comment:
Once posting a comment successfully we want to also have a message displayed telling us the post was successful such as ‘Comment added' 😃